views/administration/courses.hbs

```diff
@@ -14,11 +15,28 @@
                 <div class="container-fluid">
                     <div class="row">
                         <div class="col-sm-12 no-padding">
-                            {{> "lib/components/table" head=head body=body}}
+                            <div class="tabContainer" data-active-tab={{activeTab}}>
+                                <div class="tabs">
+                                    <a class="tab" data-tab="js-active" href="/administration/courses/?activeTab=active">
+                                        <span class="text">{{$t "courses.tab_label.currentCourses"}}</span>
+                                    </a>
+                                    <a class="tab" data-tab="js-archived" href="/administration/courses/?activeTab=archived">
+                                        <span class="text">{{$t "courses.tab_label.archivedCourses"}}</span>
+                                    </a>
+                                </div>
+                            </div>
+                            <div class="sectionsContainer"><div class="sections"><div class="section section-course" data-section="js-{{activeTab}}"></div></div></div>
```

Comment:
This empty DIVs are required to make the tab component work.

Reply:
can these empty divs being extracted to a separate component that is reused?

Reply:
I have modified the tabbar.js within https://github.com/hpi-schul-cloud/schulcloud-client/pull/2112
If these changes are ok, the component will be able to work without these empty DIVs.